lnrpc/invoicesrpc/utils.go

```diff
@@ -2,6 +2,7 @@ package invoicesrpc
 
 import (
 	"encoding/hex"
+	"errors"
```

Comment:
We commit message, we don't have a `utils` folder or package.

Reply:
fixed